patternlab/styleguide/source/_patterns/03-organisms/by-template/campaign-video.twig

```diff
@@ -0,0 +1,21 @@
+{% set textColor = campaignVideo.textColor %}
+{% set bgColor = campaignVideo.backgroundColor %}
+
+
+<section id="{{campaignVideo.id}}" class="ma__campaign-video {{ textColor ? 'ma__campaign-video--light-text' : '' }}{% if bgColor %} ma__campaign-video--{{ bgColor }}{% endif %}">
+
+  <div class="ma__campaign-video--wrapper">
+    {% set headingLevel = campaignVideo.positionTop ? 1 : 2 %}
```

Comment:
The heading level is set as a variable, `level`, to passed onto `headingLevel`.  You can see a sample in patternlab/styleguide/source/_patterns/01-atoms/04-headings/comp-heading.twig.

Can the `headingLevel` be set with the `level` value from its ascendant templates if it's not **1**?  Something like:
```
{% set headingLevel = campaignVideo.positionTop ? 1 : campaign-video.level ? 2 %}
```
It can have its default value of **2** as a fallback.

In Drupal theming page templates, the heading level numbers are defined as `level` and passed onto their descendant templates. 

So, as long as assigning the `level` value to `headingLevel` in a descendant template with its heading, the heading would have is proper heading level number based on its location in a page even the page structure changes later or the component is used in different pages.  Like seen in patternlab/styleguide/source/_patterns/01-atoms/04-headings/comp-heading.twig:
L.8: **2** is default value in case its ascendant templates are unable to pass `level`.
```
{% set headingLevel = compHeading.level ? : 2 %}
```